Mathlib/LinearAlgebra/BilinearForm/Properties.lean

```diff
@@ -413,6 +413,9 @@ theorem dualBasis_repr_apply
     (B.dualBasis hB b).repr x i = B x (b i) := by
   rw [dualBasis, Basis.map_repr, LinearEquiv.symm_symm, LinearEquiv.trans_apply,
     Basis.dualBasis_repr]
+  #adaptation_note
+  /-- Before leanprover/lean4#4814, we had a final `toDual_def` in the `rw`,
+  and no need for the `rfl`. I'm confused! -/
   rfl
```

Comment:
On master, it works with `rw [@toDual_def]` but not without the `@`

Reply:
Okay, propose we punt on this one and sort it out later.